client/landing/stepper/declarative-flow/migration/helpers/index.ts

```diff
@@ -1,25 +1,40 @@
-import { SITE_SETUP_FLOW } from '@automattic/onboarding';
+import { MIGRATION_FLOW, SITE_SETUP_FLOW } from '@automattic/onboarding';
+import { ImporterPlatform } from 'calypso/lib/importer/types';
 import { addQueryArgs } from 'calypso/lib/url';
+import { getFinalImporterUrl } from '../../internals/steps-repository/import/helper';
+import { StepperStep } from '../../internals/types';
 
-export const goToImporter = (
-	platform: string,
-	siteId: string,
-	siteSlug: string,
-	backToStep?: {
-		step: string;
-		flow: string;
+type PartialImporter = string;
+
+const isWPOrgImporter = ( importer: PartialImporter ) =>
```

Comment:
Is calling this a WP.Org Importer accurate? I wonder if calling it a WPAdmin Importer is better?

Reply:
> Is calling this a WP.Org Importer accurate?  
Yes, I renamed it to  `isWpAdminImporter`. It seems more accurate what we need.  [3ea17ce](https://github.com/Automattic/wp-calypso/pull/93700/commits/3ea17cec00be2bb0e7e2d3a8a78ad1dd09d7f412)